src/DotNetLightning.Core/Crypto/Keys.fs

```diff
@@ -53,8 +79,7 @@ and [<Struct>] PerCommitmentPoint(pubKey: PubKey) =
     member this.ToBytes(): array<byte> =
         this.RawPubKey().ToBytes()
 
-[<Struct>]
-type CommitmentSeed(lastPerCommitmentSecret: PerCommitmentSecret) =
+and [<Struct>] CommitmentSeed(lastPerCommitmentSecret: PerCommitmentSecret) =
```

Comment:
you don't need to declare CommitmentSeed with `and`, you can just move it earlier than `ChannelKeys`

Reply:
Fixed.